fs/xfstests.py.data/README

```diff
@@ -0,0 +1,49 @@
+This is a simple wrapper for running xfstests inside avocado. The steps to get
+started are really simple:
+
+1) Edit the configuration variables on the yaml file as well environment variable
+
+1.1) The variables 'TEST_DEV' and 'TEST_DIR' are mandatory and should be set to
+     a block device path and mount point path, respectively, that will be used
+     *exclusively* for xfstests. It must have the filesystem of your choice
+     previously created.
+     ex : export TEST_DEV =/dev/vda1 , export TEST_DIR=/mnt
+
+     DO NOT USE A BLOCK DEVICE WITH IMPORTANT DATA!!!
+
+
+1.2) In yaml file set test_number which test need to run  and skip_dangerous
+     (it will check with group ) and    will skip all the test .
+
+1.3) 
+    ------TODO--------------------------------------------------------------- 
```

Comment:
This section is still not clear. I can't say what I have to do with this TEST RANGE neither why there's a `-----TODO-----` line.